lib/YuiRestClient.pm

```diff
@@ -146,4 +117,38 @@ sub setup_libyui_firstboot {
     assert_script_run "chmod +x /usr/lib/YaST2/startup/Firstboot-Stage/S01-rest-api";
 }
 
+sub setup_libyui {
+    my $ip_regexp = qr/(?<ip>(\d+\.){3}\d+)/i;
+    if (is_hyperv) {
+        my $boot_timeout = 500;
+        my $svirt        = select_console('svirt');
+        my $name         = $svirt->name;
+        my $cmd          = "powershell -Command \"Get-VM -Name $name | Select -ExpandProperty Networkadapters | Select IPAddresses\"";
+        my $ip           = YuiRestClient::Wait::wait_until(object => sub {
+                my $ip = $svirt->get_cmd_output($cmd);
+                return $+{ip} if ($ip =~ $ip_regexp);
+        }, timeout => $boot_timeout, interval => 30);
+        set_var('YUI_SERVER', $ip);
+        select_console('sut', await_console => 0);
+    } elsif (check_var('BACKEND', 's390x')) {
+        select_console('install-shell');
+        my $ip = YuiRestClient::Wait::wait_until(object => sub {
+                my $ip = script_output('ip -o -4 addr list | sed -n 2p | awk \'{print $4}\' | cut -d/ -f1', proceed_on_failure => 1);
+                return $+{ip} if ($ip =~ $ip_regexp);
+                type_string('exit\n');
```

Comment:
We also can remove this line.

Reply:
Seems I accidentally reverted something. Thanks